runtime/compiler/control/J9Options.cpp

```diff
@@ -1483,10 +1486,11 @@ J9::Options::fePreProcess(void * base)
          UDATA nonPageableHowMany = 0;
          UDATA nonPageableOptionNumber = 0;
 
-         char *optionsString = NULL;
+         /* An external buffer is needed to deal with MAP_WITH_INCLUSIVE_OPTIONS. At most buffer should be sub 50 characters based off of options below */
+         char *optionsString = new char[50];
```

Comment:
What if the user passes `"-Xlp:codecache:AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"?
(Note, that's 70 "A"s)